src/components/ViewOrder.jsx

```diff
@@ -20,14 +20,28 @@ class ViewOrder extends Component {
     this.calculateTotalPrice()
   };
 
-  calculateTotalPrice = () => {
-   
+  calculateTotalPrice = () => {   
     this.state.menuData.map((item) => {
       let totalPrice = this.state.totalPrice + item.price
-      this.setState({totalPrice: totalPrice}) 
-      
+      this.setState({totalPrice: totalPrice})      
     })
   }
+
+  finalizeOrder = async () => {
+    let result = await closeOrder(this.props.orderId)
+    this.setPickUpTime(result.body.updated_at)
+  }
+
+  setPickUpTime = (timeOfOrder) => {
+    let timestamp =`${timeOfOrder.slice(11, 16)}`
```

Comment:
can you explain this guys?

Reply:
timeOfOrder is a string that contains full ISO date, yyyy-mm-dd ... etc. characters 11 to 16 have information on hours and minutes hh:mm, with slice method we read only this part of the string

Reply:
thanks good explanation